From ea9de154831ec95d7ad5ba7fda570f1d3569c7fd Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Thu, 11 Jul 2019 14:47:28 -0700
Subject: [PATCH] TRITON-1810 skip vmapi user migration test when user
 migrations are allowed globally Reviewed by: Orlando Vazquez
 <orlando@joyent.com> Approved by: Orlando Vazquez <orlando@joyent.com>

---
 test/lib/migration.js | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/test/lib/migration.js b/test/lib/migration.js
index f91e363..cfc414c 100644
--- a/test/lib/migration.js
+++ b/test/lib/migration.js
@@ -239,6 +239,13 @@ function TestMigrationCfg(test, cfg) {
     };
 
     test.bad_user_migrate_not_allowed_by_vm_default = function (t) {
+        // Check if the vmapi config allows migrations by default, if is is
+        // enabled then we can skip this test.
+        if (common.config.user_migration_allowed === true) {
+            t.ok(true, 'Skip - user migrations are enabled globally');
+            t.done();
+            return;
+        }
         // Should not be able to migrate a zone with
         // user_migration_allowed unset
         client.post({
-- 
2.21.0

